GEON-PPANG-iOS/Presentation/BakeryList/ViewControllers/Model/BakeryFilterItems.swift

```diff
@@ -33,38 +33,58 @@ struct BakeryFilterItems: Hashable {
     var filter: Filter
     var status: Status
     var leftIcon: UIImage
-    static var item: [BakeryFilterItems] = [BakeryFilterItems(filter: .HARD, status: .off, leftIcon: .disabledHardIcon),
-                                            BakeryFilterItems(filter: .DESSERT, status: .off, leftIcon: .disabledCakeIcon),
-                                            BakeryFilterItems(filter: .BRUNCH, status: .off, leftIcon: .disabledSandwichIcon)
+    static var item: [BakeryFilterItems] = [BakeryFilterItems(filter: .HARD,
+                                                              status: .off,
+                                                              leftIcon: .disabledHardIcon),
+                                            BakeryFilterItems(filter: .DESSERT,
+                                                              status: .off,
+                                                              leftIcon: .disabledCakeIcon),
+                                            BakeryFilterItems(filter: .BRUNCH,
+                                                              status: .off,
+                                                              leftIcon: .disabledSandwichIcon)
     ]
     
     public func isSelected() -> Self {
+        
         switch self.status {
         case .off:
             switch self.filter {
             case .HARD:
-                return BakeryFilterItems(filter: filter, status: status, leftIcon: .disabledHardIcon)
+                return BakeryFilterItems(filter: filter,
+                                         status: status,
+                                         leftIcon: .disabledHardIcon)
             case .DESSERT:
-                return BakeryFilterItems(filter: filter, status: .off, leftIcon: .disabledCakeIcon)
+                return BakeryFilterItems(filter: filter,
+                                         status: .off,
+                                         leftIcon: .disabledCakeIcon)
```

Comment:
p2:
요기도 이렇게 통일하면 좋을 것 같아요 ~
```suggestion
                return BakeryFilterItems(filter: filter,
                                         status: status,
                                         leftIcon: .disabledCakeIcon)
```

Reply:
ce56715f26ea26bd1678306278461c8e92fd5aec